rune/libenclave/attestation/sgx/attest.go

```diff
@@ -2,7 +2,8 @@ package sgx // import "github.com/opencontainers/runc/libenclave/attestation/sgx
 
 // RA Type
 const (
-	InvalidRaType = iota
+	UnsetRaType = iota
```

Comment:
So I don't see UnsetRaType is necessary.

Reply:
> So I don't see UnsetRaType is necessary.

Because the type of config.Enclave.RaType is `uint32` and the type of `enclave.attestation.ra_type` is string. In order to check the value of config.Enclave.RaType in `rune/libcontainer/configs/validate/validator.go` . So We need to use a uint32 value named `UnsetRaType`  to judge whether the string value `enclave.attestation.ra_type` is equal to null or not.